src/charm.py

```diff
@@ -105,6 +111,10 @@ def __init__(self, *args: Any) -> None:
         )
         self.oauth = OAuthProvider(self)
 
+        self.login_ui_endpoints = LoginUIEndpointsRequirer(
+            self, relation_name=self._login_ui_relation_name
+        )
+
         self.endpoints_provider = HydraEndpointsProvider(self)
 
         self.framework.observe(self.on.hydra_pebble_ready, self._on_hydra_pebble_ready)
```

Comment:
Since we removed `config.yaml` completely, we can also stop observing the related event :`self.framework.observe(self.on.config_changed, self._on_config_changed)`